### ISSUE TITLE
It works on Tablet but Not Phone

### ISSUE THREAD

**Issue by ramzidaher** [Martichou/rquickshare#86]
Well when I come to send from my phone the device doesn't show up directly, it would/could show up later on. But on the tablet, it shows right away no matter what.

Devices:
Samsung Tab s9fe plus
Samsung S24Ultra

**Comment by Martichou** [comment]
Hey! 

So, what I think is happening (and that's a "normal" behavior) is that the service on your desktop is advertising its availability when it starts and then periodically (through mDNS on your LAN). I also noticed that during development and I came up with a "workaround" that is to re-advertise the mDNS service when the desktop detects that a nearby device is sharing (using BLEA).

In short: Android devices send a bluetooth advertisement packet when they are sharing. The desktop gets it and re-advertise its availability. This can take up to 15s. But there's no other way to it. I can, if really needed, make the 15s customizable through config but I thought 15s wouldn't be an issue.

**Comment by ramzidaher** [comment]
Yeah its alright, thanks a lot! I love your product I wanted that!
